.travis.yml

```diff
@@ -41,8 +41,8 @@ python:
 
 before_install:
   - "mkdir /tmp/elasticsearch"
-  - "wget -O - https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/${ES_VERSION}/elasticsearch-${ES_VERSION}.tar.gz | tar xz --directory=/tmp/elasticsearch --strip-components=1"
-  - "/tmp/elasticsearch/bin/plugin install -b mapper-attachments"
+  - "wget -O - $ES_URL | tar xz --directory=/tmp/elasticsearch --strip-components=1"
+  - "if [ ${ES_URL} == ${ES2_DOWNLOAD_URL} ]; then /tmp/elasticsearch/bin/plugin install -b mapper-attachments; fi"
```

Comment:
Not needed, since full-text file indexing is not part of `invenio-record-files` anymore